pydis_site/static/css/home/index.css

```diff
@@ -218,21 +241,72 @@ h1 {
     text-align: center;
 }
 
-#sponsors .columns {
-    display: block;
-    justify-content: center;
-    margin: auto;
-    max-width: 80%;
+[data-theme="dark"] #sponsors {
+    background-color: #2C2F33 !important;
```

Comment:
The sponsors section have been revamped so they can be displayed nicely readable in dark mode. This also happens to make their sizes more consistent for light mode (see #662).

Also see [relevant discord discussion](https://discord.com/channels/267624335836053506/635950537262759947/1193170173574774855).